src/commands/basemaps-github/create-pr.ts

```diff
@@ -1,17 +1,33 @@
 import { ConfigLayer, standardizeLayerName } from '@basemaps/config';
+import { Epsg, EpsgCode } from '@basemaps/geo';
+import { fsa } from '@basemaps/shared';
 import { boolean, command, flag, oneOf, option, optional, string } from 'cmd-ts';
+import { StacCollection } from 'stac-ts';
 
 import { CliInfo } from '../../cli.info.js';
 import { logger } from '../../log.js';
 import { verbose } from '../common.js';
 import { Category, MakeCogGithub, parseCategory } from './make.cog.github.js';
 
+async function parseTargetInfo(target: string): Promise<{ name: string; title: string; epsg: EpsgCode }> {
+  const splits = target.replace('s3://', '').split('/');
```

Comment:
did we want to validate the hostname to be `linz-basemaps`?

Reply:
I think it's ok for now.

Reply:
Since this is PRing into linz/basemaps-config the only place that is valid is `linz-basemaps` it would prevent accidental import errors.

Reply:
I will make another PR to fix this.